app/_src/kubernetes-ingress-controller/gateway-api.md

```diff
@@ -4,3 +4,56 @@ type: explanation
 purpose: |
   Introduce the Kubernetes Gateway API as our preferred configuration method. Explain Gateway, GatewayClass and provide a simple HTTPRoute example
 ---
+
+Kong are proud to be a driving force behind the Kubernetes Gateway API standard. With multiple contributors in maintainer and reviewer roles, we're all-in on Gateway API as the future of Kubernetes networking.
```

Comment:
This is entirely honest, we are heavily invested in Gateway API, hold (and actively participate) in many key roles, and have been doing so since it's inception @ San Diego Kubecon in 2019.

You could probably even say more.

Reply:
```suggestion
We at Kong are proud to be a significant driving force behind the Kubernetes Gateway API standard. Our engineers helped originate the project, and we have been active as maintainers, approvers and reviewer roles from the start. We're all-in on Gateway API as the future of Kubernetes networking.
```